pkg/proxy/config/config.go

```diff
@@ -149,20 +152,22 @@ func (c *EndpointsConfig) handleDeleteEndpoints(obj interface{}) {
 	}
 	for i := range c.eventHandlers {
 		klog.V(4).Infof("Calling handler.OnEndpointsDelete")
-		c.eventHandlers[i].OnEndpointsDelete(endpoints)
+		c.eventHandlers[i].OnEndpointsDelete(endpoints, c.tenantPartitionId)
 	}
 }
 
 // ServiceConfig tracks a set of service configurations.
 type ServiceConfig struct {
-	listerSynced  cache.InformerSynced
-	eventHandlers []ServiceHandler
+	listerSynced      cache.InformerSynced
+	eventHandlers     []ServiceHandler
+	tenantPartitionId int
```

Comment:
nit - in scale-up there is no tenant partition. suggest to consider a name suited in both scale-out and scale-up.

Reply:
Keep the original as scale up can have single tenantPartitionId.